cmd/cost/get.go

```diff
@@ -0,0 +1,233 @@
+package cost
+
+import (
+	"fmt"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"log"
+	"strconv"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/costexplorer"
+	"github.com/aws/aws-sdk-go/service/organizations"
+
+	"github.com/spf13/cobra"
+
+	awsprovider "github.com/openshift/osd-utils-cli/pkg/provider/aws"
+)
+
+// getCmd represents the get command
+func newCmdGet(streams genericclioptions.IOStreams) *cobra.Command {
+	ops := newGetOptions(streams)
+	getCmd := &cobra.Command{
+		Use:   "get",
+		Short: "Get total cost of a given OU. If no OU given, then gets total cost of v4 OU.",
+		Run: func(cmd *cobra.Command, args []string) {
+
+			cmdutil.CheckErr(opsCost.complete(cmd, args))
+			org, ce, err := opsCost.initAWSClients()
+			cmdutil.CheckErr(err)
+
+			//Get Organizational Unit
+			OU := organizations.OrganizationalUnit{Id: aws.String(ops.ou)}
+			//Store cost
+			var cost float64 = 0
+
+			if ops.recursive { //Get cost of given OU by aggregating costs of all (including immediate) accounts under OU
+				getOUCostRecursive(&OU, org, ce, &ops.time, &cost)
+				fmt.Printf("Cost of %s recursively is: %f\n", ops.ou, cost)
+			} else { //Get cost of given OU by aggregating costs of only immediate accounts under given OU
+				getOUCost(&OU, org, ce, &ops.time, &cost)
+				fmt.Printf("Cost of %s is: %f\n", ops.ou, cost)
+			}
+		},
+	}
+	getCmd.Flags().StringVar(&ops.ou, "ou", "ou-0wd6-aff5ji37", "get OU ID (default is v4)") //Default OU is v4
```

Comment:
Is this default value sensitive `ou-0wd6-aff5ji37`? Is it okay to keep it here?

Reply:
@aweiteka can this information be made public?

Reply:
I would say remove it. In this case, I think there should be no default, but I'll defer to @aweiteka

Reply:
Removed. The --ou flag is now mandatory for the get, list, and reconcile commands.